web-registry/src/pages/Marketplace/Storefront/hooks/useResetErrorBanner.tsx

```diff
@@ -19,7 +19,7 @@ export const useResetErrorBanner = ({
     if (displayErrorBanner) {
       timeout = setTimeout(
         () => setDisplayErrorBanner(false),
-        DEFAULT_DURATION,
+        DEFAULT_DURATION - 1000,
```

Comment:
Is there a reason for this, or just experimenting?

Reply:
Yes before we had about that time before being able to show the error banner again once it disappeared.